modules/olm-building-operator-catalog-image.adoc

```diff
@@ -52,23 +51,19 @@ link:https://quay.io/[quay.io], tagging and pushing it to your mirror registry:
 +
 ----
 $ oc adm catalog build \
-    --appregistry-endpoint https://quay.io/cnr \
-    --appregistry-org redhat-operators \ <1>
-    --to=<registry_host_name>:<port>/<namespace>/redhat-operators:v1 <2>
+    --appregistry-org redhat-operators \//<1>
+    --to=<registry_host_name>:<port>/olm/redhat-operators:v1 \//<2>
+    [-a <path_to_registry_credentials>] \//<3>
+    [--insecure] <4>
```

Comment:
If they have followed the instructions to set up a registry I don't think they need to do an `--insecure`

Reply:
I put `-a` and `--insecure` in square brackets to show them as optional flags, but can use the word "Optional" in the callout description as well to make it more clear.